guidelines/repository-guidelines/README.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Overview
+label: internal
+---
+
+The Repository guidelines include:
+
+- The [instruction](./01-new-repository-settings.md) that lists recommended settings for a new repository in the `kyma-project` and `kyma-incubator` organizations.
+- The content desription of the [`repository-template`](https://github.com/kyma-project/community/tree/master/guidelines/repository-guidelines/repository-template) folder that provides the structure of an exemplary Kyma repository. Use it whenever you create a new repository within Kyma.
```

Comment:
```suggestion
- The content description of the [`repository-template`](https://github.com/mmitoraj/community/tree/kyma-project-io/guidelines/repository-guidelines/repository-template) folder that provides the structure of an exemplary Kyma repository. Use it whenever you create a new repository within Kyma.
```

Reply:
The link will be ok after the PR is merged.